compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -532,7 +537,10 @@ checkDiagnostics log expected got
             DRange r -> r == _range
             DSeverity s -> Just s == _severity
             DSource s -> Just (T.pack s) == _source
-            DMessage m -> standardizeQuotes(T.pack m) `T.isInfixOf` standardizeQuotes(T.unwords (T.words _message))
+            DMessage m ->
+              standardizeQuotes (T.pack m)
+                  `T.isInfixOf`
+                      standardizeQuotes (redactStablePackageIds (T.unwords (T.words _message)))
```

Comment:
instead of redacting the package ids at the end, I think setting `optDetailLevel = PrettyLevel (-1)` should get rid of them at the source - I'll submit another PR with that change once this is in